decoder/vaapidecoder_h265.cpp

```diff
@@ -1104,6 +1104,9 @@ YamiStatus VaapiDecoderH265::decode(VideoDecodeBuffer* buffer)
         NalUnit nalu;
         if (nalu.parseNaluHeader(nal, size)) {
             status = decodeNalu(&nalu);
+            if(YAMI_DECODE_INVALID_DATA == status){
```

Comment:
thanks @xuguangxin .
The other issue clips mentioned in #340 are unsupported so far,
so we just need this patch.